app/Http/Middleware/isAdmin.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Support\Facades\Auth;
+
+class isAdmin
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        if (Auth::check() && Auth::user()->isAdmin == 1) {
+            return $next($request);
+        }
+        return redirect()->back();
```

Comment:
```suggestion
        }
        
        return redirect()->back();
```
1 line trống trc return

Reply:
vẫn chưa sửa nè rm